src/components/topbar/topbar.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Form, FormGroup, Button } from 'reactstrap';
 import './topbar.scss';
 
-const TopBar = () => (
+const TopBar = ({ handleSearch }) => (
   <div className="topbar">
-
     <div className="topbar-content">
       <div className="logo">
         <img src="../../../assets/images/logo.png" alt="logo" />
       </div>
       <div className="search-form">
         <Form>
           <FormGroup>
-            <input type="text" name="search" placeholder="search for jobs " />
+            <input 
+              type="text" 
+              name="search" 
+              placeholder="search for job by title" 
+              onChange={(event) => {
+                handleSearch(event.target.value);
+              }}
+            />
             <Button type="submit" size="sm" color="primary">Search</Button>
```

Comment:
What is the purpose of the search button? It does not seem to trigger any events

Reply:
+ ⒈

Reply:
+1